CHANGELOG.md

```diff
@@ -5,6 +5,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [v0.8.0] - 2022-08-11
+
+### Changed
+
+- Redesign of the whole cogito code, based on the removal of the ofcourse package. This simplifies the code and allows to use more of the Go type system (JSON is parsed into typed Go structs, instead of string maps): many validations are done directly by the JSON parser.
+  - The behaviour should be unchanged.
```

Comment:
Hum... That line should not go in the `### Changed` section. :rofl:

Reply:
I see your point :-) To me, this is the best section (or less worst) to put it, because it is not a breaking change.
Any suggestions?

Reply:
Nope. I'm fine with keeping it there and that's why I approved the PR already.